src/transport/grpc/mod.rs

```diff
@@ -90,9 +91,9 @@ pub enum GrpcError {
     MessageHubProtoConversion(#[from] MessageHubProtoError),
 }
 
-type MessageHubClientWithInterceptor =
-    MessageHubClient<InterceptedService<Channel, NodeIdInterceptor>>;
+type MhClient = MessageHubClient<InterceptedService<Channel, NodeIdInterceptor>>;
```

Comment:
Is too short, better use `MsgHubClient`
```suggestion
type MsgHubClient = MessageHubClient<InterceptedService<Channel, NodeIdInterceptor>>;
```